console-subscriber/Cargo.toml

```
@@ -25,4 +24,4 @@ serde_json = "1"
 [dev-dependencies]
 
 tokio = { version = "^1.7", features = ["full", "rt-multi-thread"]}
-futures = "0.3"
+futures = "0.3"
```

nit: unrelated trailing whitespace change?
```suggestion
futures = "0.3"

```